src/lib/hooks/Counter.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
```

Comment:
Just adding a reminder we should remove this test component before merge. We don't need it in the library itself. It can remain until the PR audit is complete though.

Reply:
Exactly my though, left it in there so you can see for yourself if you wish to do so. It is not meant to be part of the PR.